README.md

```diff
@@ -1,25 +1,18 @@
 # geodev-hackerlabs
 
-A place to learn how to build geo apps with the ArcGIS Platform.
+> A place to learn how to build geo apps with the ArcGIS Platform.
 
 ![Steps](./geodev-hackerlabs-apps.png)
 
 ## Getting Started
 
-Follow these steps to prepare for the labs.
-
-1. Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
-2. Login to [ArcGIS Online](http://arcgis.com).
- * Search for `Geodev Hackerlabs Share` > `Search Groups`.
- * Uncheck `Only search in my account/organization`.
- * Join the group.
-3. Sign up for a GitHub account if you don't have one.
+Sign up for an [ArcGIS Developer subscription](https://developers.arcgis.com/en/sign-up/).
 
 ## Labs
 
-The labs can be completed in any order. You can use the default data but we encourage you to import and use your own data throughout. Don't forget to share your maps and apps with the [Geodev Hackerlabs Share](http://www.arcgis.com/home/group.html?owner=al_geodev&title=Geodev%20Hackerlabs%20Share) group.
```

Comment:
I suggest adding minor additional verbiage to the first sentence, for example "The labs can be completed in any order *and at your own pace*."